### PR TITLE
Images in Language Specification

### LINKED ISSUE #1566: Missing attached screen shot in Language Specification
At the end of 1.3 section of mark down version of language spec., the spec. says "...as in following screen shot" but there is no screen shot attached.

### PR DESCRIPTION
**Pull request by ahejlsberg**
Fixes #1566.

### CODE REVIEW

File: scripts/word2md.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ function convertDocumentToMarkdown(doc) {
     var tableColumnCount;
     var tableCellIndex;
     var columnAlignment = [];
+    var imageCount = 0;
```
Comment (DanielRosenwasser) [inline scripts/word2md.js]: I don't see this in the original `.ts`. Actually, why is this `.js` file even included here?

**Comment by DanielRosenwasser** [comment]
Can we remove the generated `.js` file? It seems unnecessary.

Also, where exactly are you saving the images themselves?

**Comment by yuit** [comment]
agree with @DanielRosenwasser  about removing generated javascript file.
Regarding image, I think it is in the doc folder which is where spec.md is in

File: scripts/word2md.ts [hunk 2]
```diff
@@ -258,13 +261,23 @@ function convertDocumentToMarkdown(doc: Word.Document): string {
...
+        var range = p.range;
+        var text = range.text;
         var style = p.style.nameLocal;
-        var inTable = p.range.tables.count > 0;
+        var inTable = range.tables.count > 0;
         var level = 1;
         var sectionBreak = text.indexOf("\x0C") >= 0;
 
         text = trimEndFormattingMarks(text);
+        if (text === "/") {
```
Comment (DanielRosenwasser) [inline scripts/word2md.ts]: What exactly does this mean?

**Comment by DanielRosenwasser** [comment]
I get that, I'm more curious as to what in the code exactly does that. Was extracting images out a manual process, or is it automated now? As long as we're generally confident we won't be adding more images, either are fine.

File: scripts/word2md.js (hunk 1 above)
Comment (ahejlsberg) [inline scripts/word2md.js]: I removed it again.

File: scripts/word2md.ts (hunk 2 above)
Comment (ahejlsberg) [inline scripts/word2md.ts]: Sure, I will add a comment. An inline image shows up in the text just as a "/". When I see a paragraph consisting of nothing but "/", I check to see if the paragraph contains hidden text and if so I emit that instead. The hidden text is assumed to contain an appropriate markdown image link. I could look at automatically extracting the images, but I'm not sure how much time I want to sink into further exploring the Word object model.

**Comment by DanielRosenwasser** [comment]
:+1:

**Pull request merged**